### ISSUE TITLE
Parameters tuning

### ISSUE THREAD

**Issue by yokoria99** [xuliangcs/compnet#2]
In Supplementary Material, you tested with image shape 128x128 and with differents directions.

So what if i train some dataset with image shape 224x224 or 256x256? 
Did you tested it? 
Then i need to increase the ksize as well? 

Thanks!

**Comment by xuliangcs** [comment]
Dear yokoria99, 
Sorry for not replying promptly.
- For a fair comparison, we will normalize ROI images of different sizes into single-channel `128x128` pixels. 
- We have not yet tested high-resolution images.
- The current code is only a preliminary implementation; the size of the Gabor kernel can be changed. However, the resolution of the input image cannot be changed for now, due to the fixed dimension of the fc layer (`self.fc = torch.nn.Linear(9708, 512)`)
- For different input resolutions:
    - adjust the `fc` dimensions;
    - or increase the `ksize` and convolution `stride` appropriately to keep the final flattened feature dimensions consistent.

Thanks for your suggestions.
